src/cryptojwt/jwt.py

```diff
@@ -249,10 +268,13 @@ def pack(self, payload=None, kid="", issuer_id="", recv="", aud=None, iat=None,
             else:
                 _key = None
 
-            _jws = JWS(json.dumps(_args), alg=self.alg)
+            if jws_headers is None:
+                jws_headers = {}
```

Comment:
```suggestion
            jws_headers = jws_headers or {}
```
but I agree that the old "if" is more readable for the humans!

Reply:
anyway, it would be sufficient setting as {} directly as the default value in the args and remove this code

Reply:
Done